installer/cli/microk8s.py

```diff
@@ -112,6 +112,7 @@ def _show_install_help():
       --mem      RAM in GB used by MicroK8s (default={definitions.DEFAULT_MEMORY_GB}, min={definitions.MIN_MEMORY_GB})
       --disk     Max volume in GB of the dynamically expandable hard disk to be used (default={definitions.DEFAULT_DISK_GB}, min={definitions.MIN_DISK_GB})
       --channel  Kubernetes version to install (default={definitions.DEFAULT_CHANNEL})
+      --image    Kubernetes version to install (default={definitions.DEFAULT_IMAGE})
```

Comment:
Hey @druths, thanks a lot for opening this.
Can you please correct this description for the image option?

Reply:
Woops - fixed!